Apps/AdminWebClient/src/ClientApp/src/services/restAuthenticationService.ts

```diff
@@ -75,7 +75,7 @@ export class RestAuthenticationService implements IAuthenticationService {
 
     public expireSiteMinderCookie() {
         // This expires the siteminder cookie preventing the app from login in using the cache.
-        var d = new Date();
+        const d = new Date();
         document.cookie = `SMSESSION=;domain=.gov.bc.ca;path=/;expires=${d.toUTCString()}`;
```

Comment:
In chats with the OCIO folks in the past they have said we shouldn't be altering their cookies.  Can you clarify this?

Reply:
I've removed the cookie cleanup